config-linux.md

```diff
@@ -743,6 +743,17 @@ The following parameters can be specified to set up seccomp:
     * `SCMP_ARCH_PARISC64`
     * `SCMP_ARCH_RISCV64`
 
+<!--
+The following constants are defined in the main branch of libseccomp,
+but not included in its latest release (v2.5) yet:
+https://github.com/seccomp/libseccomp/commits/main/include/seccomp.h.in
+
+* SCMP_ARCH_LOONGARCH64
+* SCMP_ARCH_M68K
+* SCMP_ARCH_SH
+* SCMP_ARCH_SHEB
+-->
```

Comment:
I created an issue and assigned it to me so that we would not forget to update it.
https://github.com/opencontainers/runtime-spec/issues/1231